common/crypto/attestation-api/test/verify_evidence_app_go/main.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// this tool is meant to be used in `$FPC_PATH/common/crypto/attestation-api/test` to ensure compatibility
+// with the shell-based attestation verification implementation in `$FPC_PATH/common/crypto/attestation-api/evidence`.
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/hyperledger/fabric-private-chaincode/internal/attestation"
```

Comment:
I am puzzled here.
The attestation API was meant to be self-contained -- and potentially be separated in its own repo.
Here there are tests which have nothing to do with the API. More precisely they follow the format, but the API is now in `internal` -- which, by the way, it's fine.